samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -0,0 +1,221 @@
+"""
+CLI command for "pipeline bootstrap", which sets up the require pipeline infrastructure resources
+"""
+import os
+from typing import Any, cast, Dict, List, Optional
+
+import click
+
+from samcli.cli.cli_config_file import configuration_option, TomlProvider
+from samcli.cli.context import get_cmd_names
+from samcli.cli.main import pass_context, common_options, aws_creds_options, print_cmdline_args
+from samcli.lib.config.samconfig import SamConfig
+from samcli.lib.pipeline.bootstrap.stage import Stage
+from samcli.lib.telemetry.metric import track_command
+from samcli.lib.utils.version_checker import check_newer_version
+from .guided_context import GuidedContext
+
+SHORT_HELP = "Sets up infrastructure resources for AWS SAM CI/CD pipelines."
+
+HELP_TEXT = """Sets up the following infrastructure resources for AWS SAM CI/CD pipelines:
```

Comment:
<img width="928" alt="image" src="https://user-images.githubusercontent.com/3804518/114949547-18eb5380-9e06-11eb-8c91-ab8fada3e035.png">

Looks like there is one empty line between each items, is it intended? It feel too "sparse"

Reply:
It is intended(henece the "\n") that is my own preference but it may change after the UX review

Reply:
I'd agree, unless there's a logical grouping between parts of text, we should avoid extra newlines (this applies especially to terminals, where real estate can be very limited).

Reply:
Let's ignore the UX issues for now; there is a UX review next week.